sdk/core/amqp-common/src/index.ts

```diff
@@ -63,3 +63,4 @@ export {
   AsyncLock,
   isNode
 } from "./util/utils";
+export { Aborter } from "@azure/core-aborter";
```

Comment:
I am not sure if we should export a type/class that we expect our user to import from its own library anyway i.e we expect users to import the `core-aborter` library
@bterlson?

Reply:
Agreed. With the changes happening with `core-aborter` I’ll update the APIs to accept an `AbortSignalLike`interface, then a customer will provide their own AbortSignal.